src/controllers/expense.controller.js

```diff
@@ -0,0 +1,99 @@
+const expensesService = require('../services/expense.service');
+const expensesHelpers = require('../helpers/expense.helpers');
+
+const get = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  const expenses = await expensesService.getExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
```

Comment:
```suggestion
const get = async (req, res) => {
  const expenses = await expensesService.getExpenses(req.query);
```
If you do nothing with variables that you get with destructing you don't need to do it